rubix-spi/src/main/thrift/bookkeeper.thrift

```diff
@@ -6,7 +6,18 @@ typedef i32 int
 enum Location {
 CACHED,
 LOCAL,
-NON_LOCAL
+NON_LOCAL,
+UNKNOWN
+}
+
+enum NodeState {
```

Comment:
We should start standardizing the indenting for code blocks in this file (either 4 spaces to match the rest of this file, or 2 spaces to match our Java code style)